site/blog/2022-11-04.mdx

```diff
@@ -0,0 +1,123 @@
+---
+slug: ecs-simplified
+title: Deploying to ECS, Simplified!
+authors: [mtp1376]
+---
+If you have ever tried to deploy your containerized application to ECS, you know that it's not going to be an easy click-to-deploy journey. In order to get your application up and running on ECS, you have to go through a bunch of resource creation. You'll need to:
+- deploy a load balancer as the point-of-contact for your app
+- create a target group for the load balancer and register the ECS tasks in it
+- add a new listener to your load balancer and connect it to the target group
+- create a security group, and you need to allow the port your app is listening on in that security group
+- attach the security group above to your load balancer
+- create a CloudWatch log group for your ECS task
+- create an ECS cluster, and definitely the task definition as well
+- Oh, and create an ECR repository to push your images to be run on the container
+
+I'm not going to continue this long list, since I've already got a headache. Doing those steps manually is going to give you a headache as well, so why bother doing all those steps yourself and risking different errors you might face when deploying your containerized app? You don't really have the time for the random IAM-related errors AWS demanding you to resolve. Besides, you already have your codebase ready and the `Dockerfile` is there, so why not just run a simple command doing something that should be simply done?
+
+In this post, we're going to discover an IaSQL module that's built to make deploying to ECS, simplified. Most of the details for deploying a container to ECS are the same (load balancers, security groups, IAM roles, etc), and we have created the `ecs_simplified` module for you so that you can give it your codebase and get your app deployed to ECS **in just 11 minutes**. All of those needed resources are going to be created automatically in your AWS account, and you'll have full access over them either through the AWS dashboard, or through your IaSQL database.
```

Comment:
I know it's not our fault, but "in just 11 minutes" feels like forever for something that *should* be able to spin up in seconds, so not sure about the emphasis on that?